pycontrails/physics/units.py

```diff
@@ -447,3 +447,26 @@ def lbs_to_kg(lbs: ArrayScalarLike) -> ArrayScalarLike:
         mass, kilograms [:math:`kg`]
     """
     return lbs * 0.45359
+
+
+def dt_to_float_seconds(
```

Comment:
I love this. I feel like this should be renamed `dt_to_seconds` with a default `dtype` of float64?

Reply:
5f96a0a772b5603c69292a3f9b7d781ef9b3114d